CHANGELOG.md

```diff
@@ -14,7 +14,7 @@
   1. [#1269](https://github.com/influxdata/chronograf/issues/1269): Add more functionality to the explorer's query generation process
   1. [#1318](https://github.com/influxdata/chronograf/issues/1318): Fix JWT refresh for auth-durations of zero and less than five minutes
   1. [#1332](https://github.com/influxdata/chronograf/pull/1332): Remove table toggle from dashboard visualization
-  1. [#1335](https://github.com/influxdata/chronograf/pull/1335): Improve UX for sanitized kapacitor settings
+  1. [#1337](https://github.com/influxdata/chronograf/pull/1337): Fix no apps for hosts false negative
```

Comment:
This was placed in the beta9 section.  Should be in the unreleased section (this is where the next release's features/bugfixes go).